packages/modal/src/ModalView.tsx

```diff
@@ -199,8 +202,11 @@ function ModalView({
   const focusTrapOptions = initialFocus
     ? {
         initialFocus: `#${id} ${initialFocus}`,
+        fallbackFocus: `#${id} .${closeClassName}`,
       }
-    : {};
+    : {
+        fallbackFocus: `#${id} .${closeClassName}`, // test fail without a fallback
```

Comment:
When I upgraded the focus trap package the modal test started failing. I kept getting an error that there were no tabbable elements. I initially tried to use [this](https://github.com/focus-trap/focus-trap-react/issues/91#issuecomment-673580519) solution but that didn't work but another solution suggested using a [fallback solution](https://github.com/focus-trap/focus-trap-react/issues/91#issuecomment-686440596) and that seemed to fix the errors